src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -453,6 +456,12 @@ export function startNewAppointmentFlow() {
   };
 }
 
+export function startNewExpressCareFlow() {
```

Comment:
I added this in order to remove the import from the appointment-list into the express-care folder, following the pattern we use for the same action related to the new appointment flow.